src/main/kotlin/org/partiql/scribe/targets/redshift/RedshiftRewriter.kt

```diff
@@ -210,148 +182,112 @@ public open class RedshiftRewriter(val onProblem: ProblemCallback) : PlanRewrite
         return super.visitRel(node, node.type)
     }
 
+    // Apply the given `excludePath` to the `bindings`
+    private fun excludeBindings(bindings: List<Rel.Binding>, excludePath: Rel.Op.Exclude.Item): List<Rel.Binding> {
```

Comment:
The following additions are pretty much the same as partiql-lang-kotlin's `PlanTyper` to compute the `StaticType` with the exclude paths applied. Additional change was adding the meta "EXPAND" to indicate a struct or a nested struct had a nested field excluded.